### PR TITLE
zh:config-adapters

### PR DESCRIPTION
**Pull request by roganw**

### CODE REVIEW

File: content_zh/docs/reference/config/policy-and-telemetry/adapters/_index.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 ---
 title: 适配器
-description: Mixer 适配器能够让 Istio 连接各种基础设施后端以完成类似指标和日志这样的功能。
+description: Mixer 适配器能够让 Istio 连接各种后端基础设施以完成类似监控指标和日志这样的功能。
 weight: 40
 type: section-index
+aliases:
```
Comment (rootsongjc) [inline content_zh/docs/reference/config/policy-and-telemetry/adapters/_index.md]: 中文页面暂时不用加alias配置，请删除

Comment (roganw) [reply content_zh/docs/reference/config/policy-and-telemetry/adapters/_index.md]: removed

**Comment by rootsongjc** [comment]
@roganw Merge the upstream master and re-lint.

**Comment by rootsongjc** [comment]
Duplicated with https://github.com/istio/istio.github.io/pull/2606
Close this PR.

**Pull request closed without merging**